.github/workflows/e2e-self-managed.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: e2e-self-managed
+
+"on":
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: ["*"]
+    tags-ignore: ["*"]
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Can you please update these to the version of the other files in `.github/workflows`, including SHA pinning and version commit at end of line. From there on, Dependabot should take after #197.

Reply:
@hiddeco done